stdlib/public/core/Builtin.swift

```diff
@@ -323,6 +323,11 @@ internal func _branchHint(_ actual: Bool, expected: Bool) -> Bool {
 @usableFromInline
 @_silgen_name("_swift_objcClassUsesNativeSwiftReferenceCounting")
 internal func _usesNativeSwiftReferenceCounting(_ theClass: AnyClass) -> Bool
+
+/// Returns the class of a non-tagged-pointer Objective-C object
+@_effects(readonly)
+@_silgen_name("swift_stdlib_getObjCClassOfAllocated")
+internal func _stdlib_classOfObjCHeapObject(_ object: AnyObject) -> AnyClass
```

Comment:
This naming inconsistency is bothersome. Otherwise seems good.

Reply:
Renamed everything to _swift_classOfObjCHeapObject. That work? :)